tensorflow/tools/ci_build/install/install_deb_packages.sh

```diff
@@ -23,6 +23,10 @@ apt-get install -y \
     build-essential \
     curl \
     git \
+    gfortran \
```

Comment:
Take a look at https://github.com/tensorflow/tensorflow/pull/1479 please. That's what I was suggesting before. Is it not better?

Should we merge that first? And use here the python-scipy and no gfortran or dev libraries?